back/engines/commercial/user_custom_fields/config/routes.rb

```diff
@@ -9,11 +9,13 @@
           resources :custom_field_options, controller: '/web_api/v1/custom_field_options' do
             patch 'reorder', on: :member
           end
+
+          resource :reference_distribution, controller: 'ref_distributions', only: %i[show create destroy]
```

Comment:
Yes, I pondered a bit about it. But then, it would also make sense to rename the model and the db tables, defeating the original goal of avoiding very lonnnng names (`UserCustomFields::Representativeness::ReferenceDistribution`). I ended up applying the following decision rule: anything that an implementation detail is written as `RefDistribution` and anything that's part of an interface with the outside world is written as `reference_distribution`. We could very much argue that the web/front-end API is external...

After having written this down, I would actually tend to rename the endpoint to ref_distribution instead of renaming the controller. (I can't help but note that as convenient as they are, rails conventions are actually encouraging us to expose implementation details in this case.)

Reply:
I do not know yet what the CL convention is, but why is it necessary to have long names? The name `user_custom_fields_representativeness_ref_distributions` is a very long table name and I wonder whether it is useful. Is it so long to express the relationship with user custom fields and the representativeness? Wouldn't `reference_distributions` be enough?

Reply:
Well, I don't think there is a guideline for that. Personally, I thought it was a reasonable thing to do to namespace the tables similarly to the active record models. I don't think it's particularly useful in this case, but I don't think it's hurting either, is it? Because in most cases, the table name is not really something you would write explicitly in the code (except for more complex queries). Is it more the case with Sequel?

But I'm not sure to understand the link between your original comment and the table name.